player/lua/console.lua

```diff
@@ -1721,12 +1723,25 @@ local function undefine_key_bindings()
     key_bindings = {}
 end
 
+local function pause_playback()
+    if not opts.pause_on_open then
+        return
+    end
+
+    was_playing = not mp.get_property_native('pause')
```

Comment:
Move it before `not opts.pause_on_open` check, in case the option is changed during console is open. We need to have `was_playing` is correct state.